app/utils/hstasklisthelper/HsTaskListHelperUtils.scala

```diff
@@ -100,6 +115,9 @@ trait HsTaskListHelperUtils extends Enumerable.Implicits {
     case EstablisherCompanyAddress | TrusteeCompanyAddress => messages("messages__schemeTaskList__sectionEstablishersCompany_add_address", _)
     case EstablisherCompanyContactDetails | TrusteeCompanyContactDetails => messages("messages__schemeTaskList__sectionEstablishersCompany_add_contact", _)
     case EstablisherCompanyDirectors => messages("messages__schemeTaskList__sectionEstablishersCompany_add_directors", _)
+    case TrusteeIndividualDetails => messages("messages__schemeTaskList__sectionEstablishersIndividual_add_details",  _)
+    case TrusteeIndividualAddress => messages("messages__schemeTaskList__sectionEstablishersIndividual_add_address",  _)
```

Comment:
can we reuse existing messages for address and contact details by giving them generic names?